pkg/test/resourcefixture/testdata/basic/privilegedaccessmanager/v1alpha1/privilegedaccessmanagerentitlement/create.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: privilegedaccessmanager.cnrm.cloud.google.com/v1alpha1
+kind: PrivilegedAccessManagerEntitlement
+metadata:
+  name: privilegedaccessmanagerentitlement-${uniqueId}
+spec:
+  projectRef:
+    external: projects/${projectId}
+  location: global
+  maxRequestDuration: 1800s
+  privilegedAccess:
+    gcpIAMAccess:
+      resourceType: cloudresourcemanager.googleapis.com/Project
+      projectRef:
+        external: projects/${projectId}
+      roleBindings:
+        - role: roles/pubsub.admin
+  requesterJustificationConfig:
+    notMandatory: {}
+  eligibleUsers:
+    - principals:
+        - serviceAccount:gsa-1-${uniqueId}@${projectId}.iam.gserviceaccount.com
```

Comment:
Actually, it can be anything. See all the available values listed here: https://cloud.google.com/iam/docs/principal-identifiers#v1

We can support a serviceAccountRef if we decide it's a reasonable improvement of the UX based on use cases and etc.